loc/npcs/conjurer.js

```diff
@@ -4,7 +4,7 @@ export default {
       en: 'Have you heard of the Crystal Shards that were discovered recently? Adventurers can use them to travel directly to distant realms. I don\'t have any Crystal Shards, but I can send you to some places since mystic forces converge at this nexus. Which realm is your desired destination?',
       ru: 'Вы слышали о Crystal Shards, которые были недавно обнаружены? Искатели приключений могут использовать их для перемещения сразу в дальние регионы. У меня сейчас нет никаких Crystal Shards, но я могу переместить вас в любое место, так как здесь сходятся мистические силы на этом нексусе. Какую зону вы желаете посетить?',
       sv: 'Har du hört talas om de Crystal Shards som upptäcktes nyligen? Äventyrare kan använda dem för att resa till fjärran riken. Jag har inga Crystal Shards, men jag kan skicka dig till vissa platser eftersom mystiska krafter samlas i denna nexus. Vilken värld vill du resa till?',
-      es: '¿Ha oído de los Crystal Shards que estaban descubierto recientemente? Los aventureros pueden usarlos para viajar directamente a tierras lejanas. No tengo ningún Crytal Shard, pero te puedo enviar a algunos lugares porque hay fuerzas místicas convergentes aquí en este nexo. ¿Adónde quiere ir?',
+      es: 'Has oído hablar de los Crystal Shards que se descubrieron recientemente?  Los Aventureros pueden usarlos para viajar directamente a reinos distantes. No tengo Crystal Shards,pero puedo enviarte a algunos lugares ya que las fuerzas místicas convergen en este nexo. ¿Qué reino es tu destino deseado?',
       da: 'Har du hørt om Crystal Shards, der blev opdaget for nylig? Eventyrere kan bruge dem til at rejse direkte til fjerne verdener. Jeg har ikke nogen Crystal Shards, men jeg kan sende dig til nogle steder, da mystiske styrker konvergerer i denne sammenhæng. Hvilket rige er din ønskede bestemmelsessted?',
```

Comment:
Please remove the whitespace before "Los", as well as removing the capital letter from "Adventureros". 
"No tengo ningún" means "I do not have" whereas your correction means "I have not". Please revert it to the original. Also change " ¿Qué reino es tu destino deseado?" to the original. It's way too elaborate, and does not match the English.

Reply:
ok